frontend/device/generic/device.lua

```diff
@@ -230,8 +230,9 @@ function Device:onPowerEvent(ev)
         local UIManager = require("ui/uimanager")
         logger.dbg("Suspending...")
         -- Mostly always suspend in Portrait/Inverted Portrait mode...
-        -- ... except when we just show an InfoMessage, it plays badly with Landscape mode (c.f., #4098)
-        if G_reader_settings:readSetting("screensaver_type") ~= "message" then
+        -- ... except when we just show an InfoMessage or when the screensaver
+        -- is disabled, as it plays badly with Landscape mode (c.f., #4098 and #5290)
+        if (G_reader_settings:readSetting("screensaver_type") ~= "message" and G_reader_settings:readSetting("screensaver_type") ~= "disable") then
```

Comment:
No real need for parens, as it's a single `and`.
(When I see parens, I expect more twisted conditions - my expectations would be deceived here :)

Reply:
I can remove the parens.  Are overly-long lines frowned upon?  Like maybe instead break the if across two lines, in which case I'd need the parens, right?

Reply:
Yep, 145 chars long is a bit too long (120 is ok) - so you can split it on 2 lines.
And still no need for parens :) look a few lines below for a multilines `and` and around to get a feel for the Lua style and simple needs.

Reply:
You could also do

```lua
local screensaver_type = G_reader_etc_long_long()
if screensaver_type ~= blah and screensaver_type ~= blahblah then -- much shorter and more legible
```

Reply:
@Frenzie:  not a big deal I can make that change but if its alright I'd prefer just to keep it consistent with with the code around it (~12 lines below with the multi-line if using the same function)

Reply:
Quite possibly because it's from me (I like long lines & parentheses, and I cannot lie 🎵).

Reply:
I went with shorter lines for the whole if branch (even for un-effected code), let me know if that's OK.

Reply:
@Frenzie: I went with a variable to replace the long function name.  Thanks for the feedback.